components/Collective.js

```diff
@@ -244,14 +241,7 @@ class Collective extends React.Component {
           `}
         </style>
 
-        <Header
-          title={collective.name}
-          description={collective.description || collective.longDescription}
-          twitterHandle={collective.twitterHandle || get(collective.parentCollective, 'twitterHandle')}
-          image={collective.image || get(collective.parentCollective, 'image') || backgroundImage}
-          LoggedInUser={LoggedInUser}
-          href={`/${collective.slug}`}
-        />
+        <Header collective={collective} LoggedInUser={LoggedInUser} href={`/${collective.slug}`} />
```

Comment:
This prop seems dead, but we have `canonicalURL` that serves this purpose

```suggestion
        <Header collective={collective} LoggedInUser={LoggedInUser} canonicalURL={`/${collective.slug}`} />
```